### ISSUE TITLE
[React Native Gallery -> Popup]: When user invokes 'Open Popup' button, keyboard focus stays on same button and narrator stays mute. 

### ISSUE THREAD

**Issue by AgneLukoseviciute** [microsoft/react-native-gallery#302]
Repro-Steps:

Launch the React Native Gallery app.
Navigate to ‘Popup’ button and invoke it.
Navigate to 'Open Popup' button and invoke it.
Listen to narrator announcement.
Actual Result:

When user invokes 'Open Popup' button, keyboard focus stays on same button and narrator stays mute. 

Expected Result:

When user invoke 'Open Popup' button, Keyboard focus should move to the opened dialog and narrator should announce about the information about it and while navigating in the dialog focus should stay within the dialog.

internal bug tracked here: https://microsoft.visualstudio.com/OS/_workitems/edit/42926599

**Comment by AgneLukoseviciute** [comment]
The issue is that keyboard focus remains on the "open popup" button. Popup should probably automatically take focus when invoked. We could workaround this by manually forcing the focus, but we should fix this in core (check that it repro's outside of gallery).

**Comment by chrisglein** [comment]
Popup may have an `autoFocus` property, although it's not shown in the docs: https://microsoft.github.io/react-native-windows/docs/popup-component
Is in the view manager: https://github.com/microsoft/react-native-windows/blob/6a6a622535f81ef0f588bade0a7efc5383fc0924/vnext/Microsoft.ReactNative/Views/PopupViewManager.cpp#L172
